build.py

```diff
@@ -64,14 +64,18 @@
 		'-I/usr/include/libxml2'
 	]
 
-# Disable until changes are merged into dispatch.
-#if "LIBDISPATCH_SOURCE_DIR" in Configuration.current.variables:
-#        foundation.CFLAGS += " "+" ".join([
-#                '-DDEPLOYMENT_ENABLE_LIBDISPATCH',
-#                '-I'+Configuration.current.variables["LIBDISPATCH_SOURCE_DIR"],
-#                '-I'+Configuration.current.variables["LIBDISPATCH_BUILD_DIR"]+'/tests'  # for include of dispatch/private.h in CF
-#        ])
-
+# Configure use of Dispatch in CoreFoundation and Foundation if libdispatch is being built
+if "LIBDISPATCH_SOURCE_DIR" in Configuration.current.variables:
```

Comment:
I don't have a ton of context on the changes in this pull request, but thought I'd chime in with a tiny nit-pick: Is there a reason you uncommented this line? That's fine of course, but Python requires `if` to contain at least one expression. To do nothing, you may use `pass`:

```py
if "LIBDISPATCH_SOURCE_DIR" in Configuration.current.variables:
    pass
```

I think if you run this as-is, it would raise a syntax error of some kind.